librz/core/project_migrate.c

```diff
@@ -131,11 +131,22 @@ RZ_API bool rz_project_migrate_v2_v3(RzProject *prj, RzSerializeResultInfo *res)
 	return true;
 }
 
+RZ_API bool rz_project_migrate_v3_v4(RzProject *prj, RzSerializeResultInfo *res) {
```

Comment:
cc @thestr4ng3r is it really necessary to have a migration to add a new field?

Reply:
Otherwise, it will fail at the unit test. Error: `RZ_PROJECT_ERR_INVALID_CONTENTS `

Reply:
Yes. In fact I think we should consider removing type links in this pr too, and make the migration convert all type links to global variables with some kind of auto-generated names.
Otherwise the next migration that does typelinks+globals => globals will be trickier because it has to take into account that an auto-generated name might already exist.

Reply:
Done, please take a look.

Reply:
Postponing is a fine option too, yes.